### ISSUE TITLE
execAsync - gets another results

### ISSUE THREAD

**Issue by ParkFramework** [m6w6/ext-pq#9]
```php
$sream = new pq\Connection;

$sream->execAsync("SELECT 'my'", function($res)
{
	print_r($res->fetchRow());
});

$sream->exec("SELECT 'another'");
```
Print:
```
Array
(
    [0] => my
)
Array
(
    [0] => another
)
```

This normal behavior?

**Comment by m6w6** [comment]
Hi! Thanks for the report.

If you mean that `exec` completes the previous `execAsync`, than I think yes.

If you mean that the callback is called twice, I'd have to investigate :)

**Comment by ParkFramework** [comment]
I mean that the callback is called twice.

**Comment by ParkFramework** [comment]
Thank you, i see a lot of work done.
When you plan to build a new version?

We started to use EventLoop based
http://docs.php.net/manual/en/class.evio.php
I use in coroutines (yield) asynchronous queries pq, well will test a new version pq :)

**Comment by m6w6** [comment]
If nothing exceptional happens, we should see a release soon.

> We started to use EventLoop based

Does [EvIO](https://php.net/evio) work with [pq\Connection::$socket](https://mdref.m6w6.name/pq/Connection#$socket)?

**Comment by ParkFramework** [comment]
> Does EvIO work with pq\Connection::$socket?
Yes, works fine, we use for notification poll, instead `socket_select`

**Comment by m6w6** [comment]
Just released 1.0.1 and 2.0.1!

Thanks!
